src/router/router.js

```diff
@@ -23,10 +23,25 @@ router.beforeEach(async (to, from, next) => {
     return next();
   }
 
-  if (needDialogRedirect && to.name !== 'Bridge') {
+  if (
+    needDialogRedirect &&
```

Comment:
Wow! May be we can add some meta-data rules about that to routes? It looks very bloat and unstable.

Reply:
needDialogRedirect or temp routes? If temp routes I will remove it when pr will be ready

Reply:
it's temporary, when pr is ready will remove it